Code/GraphMol/DistGeomHelpers/Embedder.cpp

```diff
@@ -880,10 +880,23 @@ bool setupInitialBoundsMatrix(
   unsigned int nAtoms = mol->getNumAtoms();
   if (params.useExpTorsionAnglePrefs || params.useBasicKnowledge) {
     setTopolBounds(*mol, mmat, etkdgDetails.bonds, etkdgDetails.angles, true,
-                   false, params.useMacrocycle14config);
+                   false, params.useMacrocycle14config,
+                   params.forceTransAmides);
   } else {
-    setTopolBounds(*mol, mmat, true, false, params.useMacrocycle14config);
+    setTopolBounds(*mol, mmat, true, false, params.useMacrocycle14config,
```

Comment:
There's the parameter object!

Reply:
yeah, we could probably stand to refactor this code at some point.
The one defense I'll make is that `setTopolBounds()` isn't in the public API, so it's not like it'll confuse anyone else.